### DISCUSSION TITLE
Any way to change the leader (or, prefix) key combination from Ctrl-P (which is the default) to something else to activate auto-pair's functions?

### DISCUSSION THREAD

**Discussion by rifazn** [LunarWatcher/auto-pairs#48] [Q&A] [answered]
I use `Ctrl-P` a lot for local buffer completions. Is there any way to change `Ctrl-P` being used as prefix for auto-pairs?

**Comment by LunarWatcher** [comment]
There's not really a convenient variable for switching the prefix. You'd have to change all the shortcuts manually. You can check the docs for the list as well, but the current options are `g:AutoPairsShortcutToggle`, `g:AutoPairsShortcutFastWrap`, `g:AutoPairsShortcutJump`, `g:AutoPairsShortcutBackInsert`, `g:AutoPairsShortcutIgnore`, `g:AutoPairsMoveExpression` (read the docs before remapping this one), and `g:AutoPairsShortcutToggleMultilineClose`. The code in charge of assigning the keys is [here](https://github.com/LunarWatcher/auto-pairs/blob/master/autoload/autopairs/Keybinds.vim#L222-L275), and I may have missed a few (there's a lot of variables :') )

Finding free keys is so hard these days :')

**Comment by LunarWatcher** [comment, marked as answer]
This has since been implemented (#70). It's currently live on the develop-4.0.0 branch as `g:AutoPairsPrefix`. Merger into the master branch is TBD at a Later Time:tm: when the rest of 4.0.0 is done and tested.
